### PR TITLE
Functionality to change the project name

### PR DESCRIPTION
**Pull request by DanielEkerhovd**
This PR adds functionality to change name of the project. 

Implemented changes: 

- When project name is clicked, it opens an input field.
- Inputs stops at 20.
- Input field width change with text.length
- When input field loses focus or enter is pressed, the text is changed and the p element is back with the new value

Not implemented:

- The name is not saved locally in any way

Other: 
- Deleted unnecessary elements in index.html
- Deleted extra hamburger menu icon

### CODE REVIEW

File: index.html
Comment (jonhenrikaavitsland) [inline index.html:1]: This menu icon is an intended design and should not be deleted.

File: src/js/spreadsheet/sidebar/projectName.js
Comment (jonhenrikaavitsland) [inline src/js/spreadsheet/sidebar/projectName.js:1]: We have found the figma files and perhaps we can make some changes to fit the design prototype.

https://www.figma.com/design/N7DYyZAfMzCrilVug7oiYz/Prototype?node-id=474-937&node-type=frame&t=yAAJEhTqZdtjlvJy-0

https://www.figma.com/proto/N7DYyZAfMzCrilVug7oiYz/Prototype?node-id=744-3142

**Review by jonhenrikaavitsland — CHANGES_REQUESTED** [review]
@DanielEkerhovd 
Please look into the requested changes and request a new review when it has been done. 
Thank you

File: src/js/spreadsheet/sidebar/projectName.js
Comment (sulenchy) [reply src/js/spreadsheet/sidebar/projectName.js:1]: @jonhenrikaavitsland Can you add this to the doc at `docs/yeetsheet-doc.md`

Comment (jonhenrikaavitsland) [reply src/js/spreadsheet/sidebar/projectName.js:1]: there should be a ticket to do so already @sulenchy

File: index.html [hunk 1]
```diff
@@ -260,23 +260,8 @@
...
-          <p class="text-sm uppercase">Project name</p>
-          <input type="text" class="hidden" id="projectName" />
-          <button class="sideBarBtn w-auto">
-            <span
-              data-icon="editPencil"
-              data-size="12"
-              data-class="dark:text-ys-textAndIconsLight"
-            ></span>
-          </button>
+          <p id="proName" class="text-sm uppercase">Project name</p>
```
Comment (sulenchy) [inline index.html:276]: Consider the accessibility of the p tag,

how about you try something like

```suggestion
          <p id="proName" contentEditable="true" tabindex="0" role="textbox" aria-label="Editable Project Name" class="text-sm uppercase">Project name</p>
```

**Review by sulenchy — CHANGES_REQUESTED** [review]
see my comment above

File: src/js/spreadsheet/sidebar/projectName.js
Comment (sulenchy) [reply src/js/spreadsheet/sidebar/projectName.js:1]: Pls confirm that

Comment (jonhenrikaavitsland) [reply src/js/spreadsheet/sidebar/projectName.js:1]: There is, ticket [#153](https://github.com/NoroffFEU/yeetsheet/issues/153)
@sulenchy

**Review by jonhenrikaavitsland — APPROVED** [review]

**Comment by jonhenrikaavitsland** [comment]
looks good on my end. 
good work. 
@DanielEkerhovd

**Pull request merged**